pages/news/2020-XX-XX-concrete-action.html

```diff
@@ -0,0 +1,63 @@
+title: Concrete Action: Taking a Stand with Terminology and Recruitment
+author: Mixxx, April
+date: 2020-06-08 01:00:00
+
+{% extends "post.html" %}
+
+{% block post %}
+
+{% load markup %}
+{% filter markdown:"extra" %}
+
+# &#35;BlackLivesMatter
+
+Alongside the protesters in the US and around the world, the Mixxx team unequivocally renounces police brutality.
+The systemic racism that burdens people of color every day is the responsibility of white people around the world to address.
+Mixxx' core development team feels it necessary not to remain silent on this issue, for "silence in the face of injustice is complicity with the oppressor."
```

Comment:
Is this quoting someone specific? If so, there should be attribution. If not, let's remove the quote marks.